tutorials/google-cloud/_posts/2020-09-06-oauth-2.md

```diff
@@ -146,7 +146,7 @@ Let's look at each piece in more detail.
 
 # GoogleAuthorizationCodeFlow
 
-The [AuthorizationCodeFlow](https://googleapis.dev/java/google-oauth-client/1.31.0/index.html?com/google/api/client/auth/oauth2/AuthorizationCodeFlow.html) class is the core of Google's OAuth library. You can use it to connect to any OAuth 2.0 server: [GitHub Sigin tutorial](https://blog.overops.com/tutorial-how-to-implement-java-oauth-2-0-to-sign-in-with-github-and-google/) is an example that uses it to sign into GitHub.
+The [AuthorizationCodeFlow](https://googleapis.dev/java/google-oauth-client/1.31.0/index.html?com/google/api/client/auth/oauth2/AuthorizationCodeFlow.html) class is the core of Google's OAuth library. You can use it to connect to any OAuth 2.0 server: [GitHub Sig In tutorial](https://blog.overops.com/tutorial-how-to-implement-java-oauth-2-0-to-sign-in-with-github-and-google/) is an example that uses it to sign into GitHub.
```

Comment:
You're still missing an "n" here 🙂

"Sig In" should be "Sign In"

Reply:
I don't why I keep making silly typos :man_facepalming: